GUI/src/main/kotlin/frameNavigation/FrameNavigation.kt

```diff
@@ -37,7 +37,8 @@ class FrameNavigation(state: MutableState<AppState>) : FrameNavigationInterface
     var diffBitmap: MutableState<ImageBitmap> = mutableStateOf(BufferedImage(1, 1, 1).toComposeImageBitmap())
 
     // state variables for the current frame index
-    var currentIndex: MutableState<Int> = mutableStateOf(0)
+    var currentIndex: Int = 0
+    var lock = false
```

Comment:
Can you be more specific with the naming here? What does the lock lock? Would probably easier to read if it was named something like `jumpLock` or something